PlugMe-ng/plugme-web-ui
# hotFix: optimize preloader logic

#### What does this PR do?
Optimize pre-loader indicator
#### Description of Task to be completed?
Change the logic behind the pre-loader indicator to account for every Ajax request dynamically
#### How should this be manually tested?
- Create a new content in the app
- observe the loading bar
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
N/A
#### Screenshots (if appropriate)
N/A
